sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryClientTest.java

```diff
@@ -25,6 +30,12 @@
 public class ContainerRegistryClientTest extends ContainerRegistryClientsTestBase {
     private ContainerRegistryClient client;
 
+    @BeforeAll
+    static void beforeAll() {
+        TestUtils.importImage(ImplUtils.getTestMode(), HELLO_WORLD_REPOSITORY_NAME, Arrays.asList("latest", "v1", "v2", "v3", "v4"));
```

Comment:
Since the import module is not available across the board - updated the tests to import it instead.